tobac/testing.py

```diff
@@ -327,3 +328,218 @@ def make_sample_data_3D_3blobs(data_type='iris',invert_xy=False):
         sample_data=DataArray.from_iris(sample_data)
     
     return sample_data
+
+def make_dataset_from_arr(in_arr, data_type = 'xarray'):
```

Comment:
It may be advantageous to use the pytest feature to make test dataset functions into decorators to easily allow them to be used across many test functions. Happy with this for now though

Reply:
I generally agree with this. I need to dig into that feature a bit more.